source/Outfit.cpp

```diff
@@ -196,6 +196,8 @@ void Outfit::Load(const DataNode &node)
 				if(isNewLicense(licenses, grand.Token(0)))
 					licenses.push_back(grand.Token(0));
 		}
+		else if(child.Token(0) == "unsellable")
```

Comment:
I think having user write `"unsellable"` 1 for the outfit would work too and it will automatically add itself into the dictionary (in the same way we write `"automaton"` 1 for some ships)

Reply:
Alright, I performed a force push.